src/libraries/Microsoft.Extensions.Configuration.Json/tests/ConfigurationBinderTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration.Test;
+using Xunit;
+
+namespace Microsoft.Extensions.Configuration.Json.Tests
+{
+    public class ConfigurationBinderTest
+    {
+        public class EnumConf : Dictionary<TestSettingsEnum, string>
+        {
+        }
+        public enum TestSettingsEnum
+        {
+            Option1,
+            Option2,
+        }
+        [Fact]
+        public void EnumBindCaseInsensitiveNotThrows()
```

Comment:
I think it would make more sense for this test to go in 

https://github.com/dotnet/runtime/blob/199580bedbdfe975359403b8b0b90559b5501a37/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/ConfigurationBinderTests.cs#L13

Reply:
Yeah, absolutelly right.